[2025-03-12T09:54:02.678+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table manual__2025-03-12T09:53:59.821630+00:00 [queued]>
[2025-03-12T09:54:02.694+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table manual__2025-03-12T09:53:59.821630+00:00 [queued]>
[2025-03-12T09:54:02.695+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-12T09:54:02.706+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-03-12 09:53:59.821630+00:00
[2025-03-12T09:54:02.710+0000] {standard_task_runner.py:57} INFO - Started process 67779 to run task
[2025-03-12T09:54:02.713+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'manual__2025-03-12T09:53:59.821630+00:00', '--job-id', '612', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmpnmhsx7ne']
[2025-03-12T09:54:02.714+0000] {standard_task_runner.py:85} INFO - Job 612: Subtask delete_data_from_table
[2025-03-12T09:54:02.762+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table manual__2025-03-12T09:53:59.821630+00:00 [running]> on host 1cc1501afd2a
[2025-03-12T09:54:02.858+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T09:53:59.821630+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T09:53:59.821630+00:00'
[2025-03-12T09:54:02.859+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2025-03-12' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-03-12T09:54:02.871+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-12T09:54:02.880+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-12T09:54:02.885+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2025-03-12' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-03-12T09:54:02.886+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-12T09:54:02.900+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20250312T095359, start_date=20250312T095402, end_date=20250312T095402
[2025-03-12T09:54:02.924+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-12T09:54:02.948+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
